src/transformers/modeling_ctrl.py

```diff
@@ -112,7 +112,11 @@ def forward(self, v, k, q, mask, layer_past=None, attention_mask=None, head_mask
             past_key, past_value = layer_past[0], layer_past[1]
             k = torch.cat((past_key, k), dim=-2)
             v = torch.cat((past_value, v), dim=-2)
-        present = torch.stack((k, v))
+
+        if use_cache is True:
```

Comment:
`if use_cache` clearer/more idiomatic

Reply:
I like `is use_cache is True` better because:
1) It's kinda like a type hint that `use_cache` is a boolean
2) Some code **has** to use `if use_cache is True` so that it's possible to be compiled in tensorflow, e.g. this test would fail without having the `is True`: https://github.com/huggingface/transformers/blob/7972a4019f4bc9f85fd358f42249b90f9cd27c68/tests/test_modeling_tf_common.py#L228